### PR TITLE
[enterprise-4.13] OSDOCS-5676: add lsof rn entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.13

Issue: https://issues.redhat.com/browse/OSDOCS-5676

Link to docs preview: http://file.rdu.redhat.com/jdohmann/OSDOCS-5676/release_notes/ocp-4-13-release-notes.html#ocp-4-13-lsof-rhcos

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kcarmichael08** [comment]
This looks good, but I think "lsof" shouldn't be capitalized in the heading, as it's a lower-case command. Is there a way to rewrite so that it doesn't come first? Jeana suggested "RCOS now includes lsof".

**Comment by jldohmann** [comment]
> This looks good, but I think "lsof" shouldn't be capitalized in the heading, as it's a lower-case command. Is there a way to rewrite so that it doesn't come first? Jeana suggested "RCOS now includes lsof".

this is indeed better, thanks for the suggestion @kcarmichael08 (and Jeana heh), i'll implement that change

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -74,6 +74,10 @@ To use IBM Secure Execution, you must have host keys for your host machine(s) an
 
 For more information, see xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-rhcos-using-ibm-secure-execution_installing-ibm-z-kvm[Installing {op-system} using IBM Secure Execution].
 
+[id="ocp-4-13-lsof-rhcos"]
+==== {op-system} now includes lsof
+{product-title} {product-version} now includes the `lsof` command by default in {op-system}.
```
Comment (mike-nguyen) [inline release_notes/ocp-4-13-release-notes.adoc:79]: If there's no particular reason "by default" was added, I think "now includes the `lsof` command in {op-system}" is sufficient.

**Pull request merged**